src/core/src/runtime/compute_hash.cpp

```diff
@@ -23,6 +23,10 @@
 #    include "openvino/reference/utils/registers_pool.hpp"
 #endif  // OV_CORE_USE_XBYAK_JIT
 
+#if defined(_WIN32)
+#    include <processthreadsapi.h>
+#endif
```

Comment:
```suggestion
#if __has_include(<processthreadsapi.h>)
    #include <processthreadsapi.h>
    bool may_use_dynamic_code() {
     ...
    }
#else
     constexpr bool may_use_dynamic_code() {
         return true;
    }    
#endif
```
?

Reply:
I would keep manual check if `defined(_WIN32`

`__has_include` can compile well when we don't have such header, but actually it's expected to be.
With manual checks we will have a compiler error, which is better than silent fallback to alternative branch